### PR TITLE
Update CheckboxField.js

### PR DESCRIPTION
**Pull request by twheys**
Added the name attribute for checkboxes from the redux-form input property.

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -12,8 +12,9 @@
     "storybook": "start-storybook -p 6006",
     "format": "prettier --write '**/*.js'",
     "test": "jest",
-    "test:watch": "jest --watchAll --verbose",
-    "test:coverage": "jest --verbose --coverage",
+    "test:watch": "npm run test --watchAll --verbose",
+    "test:coverage": "npm run test --verbose --coverage",
```
Comment (x8lucas8x) [inline docs/package.json]: That's lacking the -- before the args, like you did in test:update. Consequence is the coverage won't be generated. Same applies to test:watch above.

File: src/forms/CheckboxField.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const CheckboxField = ({
 }) => (
   <FormField id={input.id} help={help} meta={meta}>
     <Form.Check
+      name={input.name}
```
Comment (x8lucas8x) [inline src/forms/CheckboxField.js]: I don't see name in https://react-bootstrap.netlify.com/components/forms/#form-check-props. Am I missing something.

Comment (twheys) [reply src/forms/CheckboxField.js]: It gets passed to the `input` element. https://html.com/tags/input/

Comment (twheys) [reply src/forms/CheckboxField.js]: All of the other field components in bandit-pouch set this already. Just this one missing.

**Pull request merged**